examples/nlp/text_generation_with_miniature_gpt.py

```diff
@@ -0,0 +1,358 @@
+"""
+Title: Text Generation with miniature GPT
+Author: [Apoorv Nandan](https://twitter.com/NandanApoorv)
+Date created: 2020/05/29
+Last modified: 2020/05/29
+Description: Implement miniature version of GPT and learn to generate text.
+"""
+"""
+## Introduction
+
+This example demonstrates autoregressive language modelling using a
+a miniature version of GPT model.
+The model consists of a single transformer block with causal masking
+in the its attention layer.
+We use the text from IMDB sentiment classification dataset for training
+and generate new movie reviews for a given prompt.
+When using this script with your own data, make sure it has atleast
+1M words.
+
+**References:**
+
+- [GPT](https://s3-us-west-2.amazonaws.com/openai-assets/research-covers/language-unsupervised/language_understanding_paper.pdf)
+- [GPT-2](https://d4mucfpksywv.cloudfront.net/better-language-models/language_models_are_unsupervised_multitask_learners.pdf)
```

Comment:
Are these 2 papers on ArXiv? If so, link to the abstract on ArXiv.

Reply:
These are not on arxiv. They are there on semantic scholar though